FlashCards.AhmadJer99/FlashCards/Dtos/CardDto.cs

```diff
@@ -0,0 +1,9 @@
+﻿
```

Comment:
🟠 **Code Style**

💡 No blank lines at start of file.

Reply:
Done fixed most of the violated coding conventions!